lib/balrog/generators/view_generator.rb

```diff
@@ -0,0 +1,26 @@
+class Balrog::ViewGenerator < Rails::Generators::Base
+
+  desc "Copies the Balrog gate view and layout into your application, where you can edit and style them."
+
+  def copy_gate_view
+    gate_view = File.open(
+      File.join(__dir__, '../../../', 'app/views/balrog/gate.html.erb'),
+      'r')
+
+    content = gate_view.read
+    gate_view.close
```

Comment:
I'm not too sure about this, but wouldn't you want to close the file *after* you've used the content on the next line?